src/ipc/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2023-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+include(GenerateMPCode)
+generate_from_capnp(capnp/echo.capnp)
+generate_from_capnp(capnp/init.capnp)
+
+add_library(bitcoin_ipc STATIC EXCLUDE_FROM_ALL
+  ${CMAKE_CURRENT_BINARY_DIR}/capnp/echo.capnp.c++
+  ${CMAKE_CURRENT_BINARY_DIR}/capnp/echo.capnp.proxy-client.c++
+  ${CMAKE_CURRENT_BINARY_DIR}/capnp/echo.capnp.proxy-server.c++
+  ${CMAKE_CURRENT_BINARY_DIR}/capnp/echo.capnp.proxy-types.c++
+  ${CMAKE_CURRENT_BINARY_DIR}/capnp/init.capnp.c++
+  ${CMAKE_CURRENT_BINARY_DIR}/capnp/init.capnp.proxy-client.c++
+  ${CMAKE_CURRENT_BINARY_DIR}/capnp/init.capnp.proxy-server.c++
+  ${CMAKE_CURRENT_BINARY_DIR}/capnp/init.capnp.proxy-types.c++
```

Comment:
In commit "cmake: Add MULTIPROCESS option " (15ce0774f84281abff008a30890ac9ca58417ed8)

Would be nice if generate_from_capnp macro could return these generate filenames since it is already passing the same ones to `add_custom_command` Or maybe there is a way to retrieve the generated filenames from the custom command target.

Reply:
Thanks! [Reworked](https://github.com/hebasto/bitcoin/pull/118#issuecomment-1989503059).